aspnetcore/choose-aspnet-framework.md

```diff
@@ -19,28 +19,27 @@ No matter the web application you are creating, ASP.NET has a solution for you:
 
 ## ASP.NET Core
 
-ASP.NET Core is a new open-source and cross-platform .NET framework for building modern cloud-based web applications on Windows, Mac, or Linux.
+ASP.NET Core is an open-source, cross-platform framework for building modern, cloud-based web applications on Windows, macOS, or Linux.
 
 ## ASP.NET
 
-ASP.NET is a mature web platform that provides all the services that you require to build enterprise-class, server-based web applications using .NET on Windows.
+ASP.NET is a mature framework that provides all the services needed to build enterprise-class, server-based web applications on Windows.
 
 ## Which one is right for me?
 
 | ASP.NET Core | ASP.NET |
 |---|---|
-|Build for Windows, Mac, or Linux|Build for Windows|
-|Use [MVC](mvc/overview.md) or [Web API](tutorials/first-web-api.md)|Use [Web Forms](https://docs.microsoft.com/aspnet/web-forms), [SignalR](https://docs.microsoft.com/aspnet/signalr), [MVC](https://docs.microsoft.com/aspnet/mvc), [Web API](https://docs.microsoft.com/aspnet/web-api/), or [Web Pages](https://docs.microsoft.com/aspnet/web-pages)|
+|Build for Windows, macOS, or Linux|Build for Windows|
+|Use [MVC](xref:mvc/overview) or [Web API](xref:tutorials/first-web-api)|Use [Web Forms](https://docs.microsoft.com/aspnet/web-forms), [SignalR](https://docs.microsoft.com/aspnet/signalr), [MVC](https://docs.microsoft.com/aspnet/mvc), [Web API](https://docs.microsoft.com/aspnet/web-api/), or [Web Pages](https://docs.microsoft.com/aspnet/web-pages)|
 |Multiple versions per machine|One version per machine|
-|Develop with Visual Studio or Visual Studio Code using C#|Develop with Visual Studio using C#, VB or F#|
-|New platform|Mature platform|
+|Develop with Visual Studio, [Visual Studio for Mac](https://www.visualstudio.com/vs/visual-studio-mac/), or [Visual Studio Code](https://code.visualstudio.com/) using C# or F#|Develop with Visual Studio using C#, VB, or F#|
 |Ultra performance|High performance|
```

Comment:
@danroth27 Thoughts? What if we linked to recent TechEmpower benchmark results for ASP.NET Core? "Ultra performance" doesn't mean anything to me.